pyEpiabm/pyEpiabm/sweep/initial_household_sweep.py

```diff
@@ -439,7 +440,7 @@ def __call__(self, sim_params):
                         elif len(household.persons) == 2:
                             self.two_person_household_ages(household.persons)
 
-                        else:
+                        elif len(household.persons) >= 3:
```

Comment:
Is there a final else clause if this isn't hit either? or should that have already hit the valueerror above?

Reply:
Hmm maybe still good to have a check here just incase - add an else clause with a similar logging warning if it encounters an empty household